agent/render/watch/option.go

```diff
@@ -23,4 +25,7 @@ func (w *WatchOptions) Init() {
 	if w.MaxRecords <= 0 {
 		w.MaxRecords = 100
 	}
+	if w.JsonOutput && w.JsonOutputFile == "" {
+		w.JsonOutputFile = "/tmp/kyanos_watch.json"
```

Comment:
This sounds like a bad idea. using filepath.Join(os.TempDir(), "kyanos_watch.json") a better option

But then, I would like to suggest having only one argument

--json-output would dump to stdout
--json-output=/path/file to a file

So no json-output-file option would be added

Reply:
Sounds reasonable,I will do some modify later.

Reply:
> This sounds like a bad idea. using filepath.Join(os.TempDir(), "kyanos_watch.json") a better option
> 
> But then, I would like to suggest having only one argument
> 
> --json-output would dump to stdout --json-output=/path/file to a file
> 
> So no json-output-file option would be added

I agree with you.

In fact, I was wondering if we could add a `--format` option as well, change `--json-output` to `--output`? Because maybe the format is not limited to json. But I think this is OK, can be simple to only support json.

Reply:
I would say output is enough for now, then if there is a need output-format could be added.